tekton-pipelines/base/tasks/buildpack-task.yaml

```diff
@@ -85,6 +84,93 @@ spec:
           mountPath: /cache
       securityContext:
         privileged: true
+    
+    # I dislike this hack 
+    # https://github.com/tektoncd/catalog/blob/main/task/buildpacks/0.3/buildpacks.yaml#L68-L113
+    # but we should use it until this is resolved:
+    # - https://github.com/buildpacks/tekton-integration/issues/33
+    # - https://github.com/buildpacks/tekton-integration/issues/32
+    # TODO: check in the future and adjust when project.toml is clear to be supported.
+  
+    - name: set-env-vars
```

Comment:
Why do we need such a step? Is it used in other steps?

Reply:
discussed at the devops meeting in details.